test/parallel/test-crypto-cipher-decipher.js

```diff
@@ -25,7 +25,7 @@ function testCipher1(key) {
   let txt = decipher.update(ciph, 'hex', 'utf8');
   txt += decipher.final('utf8');
 
-  assert.strictEqual(txt, plaintext, 'encryption and decryption');
+  assert.strictEqual(txt, plaintext, `${txt} is encrypted, ${plaintext} is unencrypted`);
```

Comment:
I personally would prefer to just remove the error message overall instead of using the individual ones in all these cases. The reason is that it could theoretically be a different case than the description value (the test should pass and we do not know the real reason why the test would not pass).

Reply:
Are you saying in every case (within this file) to remove message? Oh I see, I forgot that the message at the end is outputted when the assertion fails not when it passes.

Reply:
I agree with @BridgeAR. I think it's better to just remove the custom messages (third argument) here and use the default one.